dll/win32/shlwapi/ordinal.c

```diff
@@ -2732,12 +2754,41 @@ DWORD WINAPI SHGetRestriction(LPCWSTR lpSubKey, LPCWSTR lpSubName, LPCWSTR lpVal
  *  different POLICYDATA structure and implements a similar algorithm adapted to
  *  that structure.
  */
+#ifdef __REACTOS__
+DWORD WINAPI
+SHRestrictionLookup(
+    _In_ DWORD policy,
+    _In_ LPCWSTR key,
+    _In_ const POLICYDATA *polTable,
+    _Inout_ LPDWORD polArr)
+#else
 DWORD WINAPI SHRestrictionLookup(
 	DWORD policy,
 	LPCWSTR initial,
 	LPPOLICYDATA polTable,
 	LPDWORD polArr)
+#endif
 {
+#ifdef __REACTOS__
+    size_t iItem = 0;
+    const POLICYDATA *entry;
+
+    TRACE("(0x%08lX, %s, %p, %p)\n", policy, debugstr_w(key), polTable, polArr);
+
+    if (!polTable->appstr)
```

Comment:
This is not needed because this is caught by the for-loop. And you'll then get automatically the trace message that is at the end of the function.